google/resumable_media/_download.py

```diff
@@ -162,10 +185,15 @@ def _process_response(self, response):
         self._finished = True
         _helpers.require_status_code(
             response, _ACCEPTABLE_STATUS_CODES, self._get_status_code)
+        if self._stream is not None:
+            self._write_to_stream(response)
```

Comment:
This is an iffy proposition (at least as far as #5 is concerned).

`_process_response` is meant to be sans-I/O, but we'd like `requests.Download._write_to_stream` to override the default behavior and consume the content in chunks (still a single GET, just HTTP chunking). But this would be network I/O, so would belong in `consume()`, not here.